examples/flutter_gallery/lib/gallery/demos.dart

```diff
@@ -9,8 +9,10 @@ import 'icons.dart';
 
 class GalleryDemoCategory {
   const GalleryDemoCategory._({ this.name, this.icon });
-  @required final String name;
-  @required final IconData icon;
+  @required
```

Comment:
The annotations should be removed from the class members here and instead be added to the named arguments in the constructor.

Reply:
done